### ISSUE TITLE
Unable to rotate text 

### ISSUE THREAD

**Issue by rohaanhamid** [unvell/ReoGrid#180]
I am using version 2.1 from Nuget and I cannot rotate text. The issue is similar to the one reported in #72 and #73 

The RotateAngle Property has not been renamed either.

My setup is as follows:
Nuget Package Version: 2.1.0
Platform: WPF

**Comment by Gnuelmpf** [comment]
Hi rohaanhamid,
this works fine for  me:

![image](https://user-images.githubusercontent.com/43831414/46733504-d5da2380-cc90-11e8-8835-4813053bd128.png)

![image](https://user-images.githubusercontent.com/43831414/46733553-f1452e80-cc90-11e8-8d1f-64954e5232c5.png)

**Comment by rohaanhamid** [comment]
Are you using 2.1.0 it in a WPF application? The RotateAngle property was supposed to be changed to RotationAngle based on #73

**Comment by Gnuelmpf** [comment]
You didn't pointed out that. 
:)
I use a plain 'Windows Form App (.NET)'

**Comment by rohaanhamid** [comment]
@Gnuelmpf Sorry about that. I should have added more detail.

Anyways, tried the WinForms version of this library and I can confirm that text rotation works as expected.

**Comment by jingwood** [comment]
The WPF Text rotation was not supported before, now I have added a simple implementation that can support the text rotation from latest code.

For example:
```csharp
var cell = grid.CurrentWorksheet.Cells["D12"];
cell.Style.RotationAngle = 90;
```
Will get this:
<img width="546" alt="wpf text rotation" src="https://user-images.githubusercontent.com/3468046/48951281-4f984900-ef81-11e8-92e4-56f6c126d281.png">

Currently I just added a simple support that rotated text will only aligned to cell center, that means always vertical and horizontal center alignment. I will consider to improve it in future.
